tests/telemetry/test_telemetry_metrics_e2e.py

```diff
@@ -89,14 +89,27 @@ def test_telemetry_metrics_enabled_on_gunicorn_child_process(test_agent_session)
         response_content = json.loads(response.content)
         assert response_content["telemetry_metrics_writer_running"] is True
         assert response_content["telemetry_metrics_writer_worker"] is True
-        assert response_content["telemetry_metrics_writer_queue"][0]["metric"] == "test_metric"
-        assert response_content["telemetry_metrics_writer_queue"][0]["points"][0][1] == 3.0
+        assert response_content["telemetry_metrics_writer_queue"][0]["metric"] == "datadog.span_created"
+        assert response_content["telemetry_metrics_writer_queue"][0]["points"][0][1] == 32.0
```

Comment:
Don't love this number... 32? I'd guess there would be some number of spans created per request, however I'm not sure how that number would be divisible by 3(the number of requests being made)

Reply:
Same with below, 2 requests, 14 spans? 7 per request? not sure. I may need to change where I call `telemetry_metrics_writer.add_count_metric("tracers", "datadog.span_created")`

Reply:
figured out that this way of tracking span creation wasn't very good since we literally look at the metrics mid-trace.